### ISSUE TITLE
bootchart spams console in VM (cannot read disk model)

### ISSUE THREAD

**Issue by ikeydoherty** [systemd/systemd#461]
Currently when bootchart is enabled, an error is printed to the console, despite being a non-fatal condition.

Relevant "fix" would be:

```
diff --git a/src/bootchart/svg.c b/src/bootchart/svg.c
index 0132475..c6c1a84 100644
--- a/src/bootchart/svg.c
+++ b/src/bootchart/svg.c
@@ -173,8 +173,6 @@ static int svg_title(FILE *of, const char *build, int pscount, double log_start,
                 snprintf(filename, sizeof(filename), "/sys/block/%s/device/model", rootbdev);

                 r = read_one_line_file(filename, &model);
-                if (r < 0)
-                        log_warning("Error reading disk model for %s: %m\n", rootbdev);
         }

         /* various utsname parameters */
```

Is this something that would be fixed? If so I'll follow up with a pull req.

**Comment by ikeydoherty** [comment]
Or he'll do it before I post this.

**Comment by zonque** [comment]
Fixed in 4e5356169a
